app/src/main/java/io/homeassistant/companion/android/widgets/entity/EntityWidget.kt

```diff
@@ -30,6 +29,8 @@ class EntityWidget : BaseWidgetProvider() {
         internal const val EXTRA_TEXT_SIZE = "EXTRA_TEXT_SIZE"
         internal const val EXTRA_STATE_SEPARATOR = "EXTRA_STATE_SEPARATOR"
         internal const val EXTRA_ATTRIBUTE_SEPARATOR = "EXTRA_ATTRIBUTE_SEPARATOR"
+
+        private var lastResolvedTextSuccess = true
```

Comment:
This is like making the variable a static variable by putting it in the companion object.  By putting it here all Entity Widgets could show failed even if just one fails.

Reply:
In testing I couldn't get one failure to change any other widgets, but I see how it can theoretically happen. Changed to return the information from the function.